src/lib/viewers/controls/media/DurationLabels.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import './DurationLabels.scss';
+
+export type Props = {
+    currentTime?: number;
+    durationTime?: number;
+};
+
+export function formatTime(time: number): string {
+    const hours = Math.floor(time / 3600);
+    const minutes = Math.floor((time % 3600) / 60);
+    const seconds = Math.floor((time % 3600) % 60);
+    const hour = hours > 0 ? `${hours.toString()}:` : '';
+    const sec = seconds < 10 ? `0${seconds.toString()}` : seconds.toString();
+    let min = minutes.toString();
+
+    if (hours > 0 && minutes < 10) {
+        min = `0${min}`;
+    }
```

Comment:
Could this be moved to a single line like:
```suggestion
    const min = hours > 0 && minutes < 10 ? `0${minutes.toString()}` : minutes.toString()
```

Reply:
Sure.